### ISSUE TITLE
ESC-Real

### ISSUE THREAD

**Issue by Phhofm** [dslisleedh/ESC#1]
Nice, thank you for your work and for the code release (that was fast) and for all the official pretrain models :)

Im writing here because im also interested in the real world scenario.

So my questions would be:
1. If you are planning to incorporate esc-real as an option in the current code (as far as I understand, its a bit trickier than just doubling N from esc, but also incorporating three layers of convolutions as S for example?).
2. If you plan on releasing the esc-real model

In any case, thank you, im excited to try this network out :)

**Comment by dslisleedh** [comment]
Thank you for your interest in our work, @Phhofm  !

1. Yes, I added the ESC-real code to the repository. Actually, in the ESC-Real, LayerNorms are added before ConvFFNs just like in ESC-FP. But I forgot to add that detail in the paper. I'll update it in the next revision.
2. I do plan to add the Pixel-wise Loss model and GAN Loss model for ESC-Real, but it might take some time. Since I'm not very experienced with RealWorld SR tasks, I'm not entirely sure if the current results are optimal. I'll upload them after some more training.

Please let me know if you have any additional questions. Thank you.

**Comment by Phhofm** [comment]
Thats great, thank you :)
Concerning the models thats fine, no stress. I might try training my own pretrain using my [bhi_small](https://huggingface.co/datasets/Phips/BHI_small) (or [bhi_full](https://huggingface.co/datasets/Phips/BHI)) dataset.

Thank you :)
